app/_hub/kong-inc/application-registration/index.md

```diff
@@ -122,11 +122,96 @@ params:
         should last, after which the client will need to refresh the token.
         Set to `0` to disable the expiration.
   extra: |
-    When configuring the plugin, at least one of the following OAuth2
-    auth flows must be enabled:
+    **Important:** When configuring the plugin, at least one of the following
+     OAuth2 auth flows must be enabled:
     - Authorization Code
     - Client Credentials
     - Implicit Grant
     - Password Grant
 
 ---
+
+## Examples
+
+Replace `localhost` with your hostname, `{service}` with your Service name,
```

Comment:
Since this is a beta feature and therefore not prod enviro, I figured it was ok to use localhost explicitly.

Reply:
In the docs, we normally use <DNSorIP> or <admin-hostname>, see: https://docs.konghq.com/enterprise/1.5.x/deployment/installation/docker/#step-6-start-kong-enterprise-with-kong-manager-and-kong-developer-portal-enabled

Reply:
Will do, thank you

Reply:
Note that the generated examples from the plugin template use "kong", so this is inconsistent. I'll add this to a list of issues with the template.

`curl -X POST http://kong:8001/services/{service}/plugins`

Reply:
> In the docs, we normally use `<DNSorIP>` or `<admin-hostname>`, see: https://docs.konghq.com/enterprise/1.5.x/deployment/installation/docker/#step-6-start-kong-enterprise-with-kong-manager-and-kong-developer-portal-enabled

Step 7 in the doc you referenced shows `localhost`

![image](https://user-images.githubusercontent.com/3756245/82070796-dd4a8580-969a-11ea-8b5b-9d72f3010a1d.png)

Reply:
Inc. in commit 491a18d

Reply:
Yep, that's an oversight and shouldn't be there.

Reply:
approved your pr, thanks!